.gitignore

```diff
@@ -112,4 +112,5 @@ credentials.json
 
 # League
 league/
-settings.json
\ No newline at end of file
+settings.json
+/autoleagueplay/bots/
```

Comment:
I would not recommend using the autoleagueplay folder as your working directory for league play (I am assuming that is why you had to add this). That will get annoying since ALP creates a lot of files. Either use `league/` which is already ignored or a totally different folder.